Testing/airport.spec.js

```diff
@@ -0,0 +1,252 @@
+const { assertEquals } = require(`./testing-framework`);
+const Airport = require(`../src/Airport`);
+const Airplane = require(`../src/Airplane`);
+
+let expected;
+let actual;
+let result;
+let airplane;
+let airport;
+let weather;
+
```

Comment:
I think I should have added a tenth test to check that a plane can't fly when stationed at another airport. In a sense, I felt that it was unnecessary since test 9 necessarily precludes test 10 not working but it would have been good practice to include anyway.